.npmrc

```diff
@@ -0,0 +1,4 @@
+registry=https://registry.npmjs.org/
+@zowe:registry=https://registry.npmjs.org/
```

Comment:
On some of the other projects, the .npmrc is configured to pull Zowe CLI from Zowe's Artifactory server to prevent metric inflation on NPM. Is that something that we should consider doing here?

Reply:
I believe @zFernand0 suggested placing the `.npmrc` this way, Fernando did you have a reason to pull from this registry instead of the way @awharn  is suggesting?

Reply:
Either one would work, as long as the Integrity checksums match what we get back from either registry.
My problem was mostly due to mismatched checksums.
I kind of agree with Andrew's point about metric inflation on NPM though. 😋